build/initialize-pipeline.ps1

```diff
@@ -0,0 +1,84 @@
+param (
+  [string]$connectionString = ""
+)
+
+function AcquireLease($blob) {
+  try {
+    return $blob.ICloudBlob.AcquireLease($null, $null, $null, $null, $null)    
+  } catch {
+    Write-Host "  Error: $_"
+    return $null
+  } 
+}
+
+# parse PR title to see if we should pack this
+$response = Invoke-RestMethod api.github.com/repos/$env:BUILD_REPOSITORY_ID/pulls/$env:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER
+$title = $response.title.ToLowerInvariant()
+Write-Host "Pull request '$title'"
+if ($title.Contains("[pack]")) {
+  Write-Host "##vso[task.setvariable variable=BuildArtifacts;isOutput=true]true"
+  Write-Host "Setting 'BuildArtifacts' to true."
+}
+
+# set the build suffix
+$branchName = $env:BUILD_SOURCEBRANCHNAME
+$buildReason = $env:BUILD_REASON
+$buildNumber = $env:BUILD_BUILDID
+Write-Host "Branch name: '$branchName'"
+Write-Host "Build reason: '$buildReason'"
+Write-Host "Build number: '$buildNumber'"
+
+$suffix = "ci"
+$emgSuffix = "ci$buildNumber" #ExtensionsMetadataGenerator suffix
+
+if ($branchName -eq "release/3.0") {
+  $suffix = ""
+  $emgSuffix = ""
+}
```

Comment:
nit: can be simplified? 

if ($branchName -ne "release/3.0") {
 $suffix = "ci"
 $emgSuffix = "ci$buildNumber"
}

Reply:
What piece would you like simplfied? We can come back and fine-tune this, certainly.

It's a bit annoying that our versioning of packages is so different or I wouldn't have to create this second "emgSuffix" property for the ExtensionsMetadataGenerator (I haven't liked how every build produces "1.1.3"... it's tough to know what version you're actually using). This was the quickest "fix" I could come up with for that.